### DISCUSSION TITLE
Is Atom still slow?

### DISCUSSION THREAD

**Discussion by trusktr** [atom/atom#23651] [General]
The main reason I've been in VS Code is speed. I love Atom's UX, but I felt I was spending too much time loading new Atom windows, etc. 

Any improvements lately?

**Comment by keevan** [comment]
In my own experience, the opening speed of Atom vs VSCode, is similar. However, _most_ of the time VSCode has a slightly faster _time-to- respond/do-things_ . That said, if you love the experience of one, can perform the things you need to daily, and the only issue seems to be window opening speed, perhaps can take a look at some packages (project manager comes to mind) which help you get where you need to, without opening new Atom windows, i.e. switch between projects, files, etc. without needing to open a new window. 

This is however dependent on your circumstances, and I'm not currently aware of why you might need to open new Atom windows, but I'm just throwing some ideas out :smile: 

Also, I know Atom has a **Time Cop** feature added by default. You can open this up via the command palette, and choose `Timecop: View`. This should show you where all if not most of the time is spent, and usually is caused by packages that are either slow, or you don't actually need, so you might be able to go through and disable them, such that the next time you open Atom it's not _as slow_.

I'm not aware of a similar package available in VSCode - if anyone wants to chime in on that end.

That said, it would be an _absolute treat_ if Atom or VSCode or any editor opens as fast as Sublime, but has all the features you need ready at hand. :rocket:

**Comment by vwallajabad** [comment]
You can actually speed up atom by disabling packages. 
1. Go to settings  
2. Click "Install a Package" 
3. Click "Open installer"
4. Click "packages"
Click packages you don't use often or at all see the "This package added ... to startup time". This way you can see what packages take a long time to load and remove them. This way you can speed up atom.
